steward/src/commands/cork_proposal/cellar_v1/set_fees_distributor.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{application::APP, cellars, commands::cork_proposal::print_proposal, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use deep_space::Address;
+use steward_proto::steward::{
+    cellar_v1_governance::{Function, SetFeesDistributor},
+    governance_call::Call,
+    CellarV1Governance, GovernanceCall,
+};
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nCalls setFeesDistributor() on the target cellar contract at the specified block height.\nFor more information see https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol"
+)]
+pub struct SetFeesDistributorCmd {
+    #[clap(short, long)]
+    /// Fee distributor's address
+    new_fees_distributor: Address,
+
+    /// Target contract for scheduled cork.
+    #[clap(short, long)]
+    cellar_id: String,
+
+    /// Block height to schedule cork.
+    #[clap(short, long)]
+    block_height: u64,
+
+    /// Only print JSON output, omitting explanatory text
+    #[clap(short, long)]
+    quiet: bool,
+}
+
+impl Runnable for SetFeesDistributorCmd {
+    fn run(&self) {
+        abscissa_tokio::run_with_actix(&APP, async {
+            cellars::validate_cellar_id(&self.cellar_id)
+                .await
+                .unwrap_or_else(|err| {
+                    status_err!("invalid cellar ID: {}", err);
+                    std::process::exit(1);
+                });
+
+            if self.new_fees_distributor.get_prefix().ne("somm") {
```

Comment:
Nit: I believe the full qualified prefix is `somm1`, which indicates the end of the prefix.

Reply:
Mm I'll double check by running the command but I'm pretty sure the 1 is ignored when retrieving prefixes in deep_space; it definitely is in cosmrs (which ocular uses).